sdks/dart/client-dart-sdk/example/web_example/README.md

```diff
@@ -1,16 +1,9 @@
-# web_example
+# A Flutter Example
 
-A new Flutter project.
+This is a simple Flutter example and it uses the GET api to ensure the radio is kept on.
```

Comment:
This is a Flutter Web example which demonstrates FeatureHub Dart SDK implementation using GET api request to get feature states from the FeatureHub repository.